common/v2/components/TransactionFlow/index.ts

```diff
@@ -1,2 +1,13 @@
-export { default as ConfirmTransaction } from './ConfirmTransaction';
-export { default as TxReceipt } from './TxReceipt';
+import { withProtectTransaction } from '../../features/ProtectTransaction/components';
+import SignTransaction from '../../features/SendAssets/components/SignTransaction';
+
+import { default as ConfirmTransaction } from './ConfirmTransaction';
+import TxReceipt from './TxReceipt';
+
+const TxReceiptWithProtection = withProtectTransaction(TxReceipt, SignTransaction);
```

Comment:
Yeah exactly this issue appear when I try to include SignTransaction within, because it creates circular dependency. Because I need to wrap SignTransaction component with Protect Tx, thus creating a circular dependency. 

Calling HOC without export, caused some weird issues, when component particular SendAssetsForm on rerender was reset to initial state. Exporting it fixed this issue.